configure-buildfarm/tasks/main.yml

```diff
@@ -40,7 +45,7 @@
 - debug: var=android_slave_image_def
 
 - name: "Configure pod template"
-  shell: "java -jar /tmp/jenkins-cli.jar -s https://{{ route_output.stdout }}  groovy ../roles/configure-buildfarm/files/podtemplate-config.groovy {{ android_slave_image }}"
+  shell: "java -jar /tmp/jenkins-cli.jar -s https://{{ route_output.stdout }}  groovy  configure-buildfarm/files/podtemplate-config.groovy {{ android_slave_image }}"
```

Comment:
This change will affect the productised version. Can we use similar logic with rhmap_version to decide which dir path is used?